druid-shell/src/hotkey.rs

```diff
@@ -112,7 +112,7 @@ impl HotKey {
     /// [`KeyboardEvent`]: keyboard_types::KeyEvent
     pub fn matches(&self, event: impl Borrow<KeyEvent>) -> bool {
         let event = event.borrow();
-        self.mods == event.mods && self.key == event.key
+        self.mods == (event.mods & (!Modifiers::NUM_LOCK)) && self.key == event.key
```

Comment:
This doesn't cover the case of CAPS_LOCK or other fancy modifiers either. What I recommend is doing `|` over the four basic modifiers: shift, ctrl, alt, meta.

I had this on my radar to address as part of the rework, but missed it. Thanks for finding it, and sorry.

Reply:
Indeed I miss some other cases, I have fixed it by now, but I cannot add shift and ctrl modifier as they are used to manage the behavior from text selection/movement in text_input.rs line 67 in handle_event.

Reply:
Hum now I failed some doctest, I'll check it later. Maybe it's a too simplistic fix.